### DISCUSSION TITLE
air-gapped Offline deployment guide

### DISCUSSION THREAD

**Discussion by busyboy77** [kube-logging/logging-operator#1547] [Q&A] [answered]
> Bugs should be filed for issues encountered whilst operating logging-operator.
> You should first attempt to resolve your issues through the community support
> channels, e.g. Slack, in order to rule out individual configuration errors. [#logging-operator](https://slack.banzaicloud.io/)
> Please provide as much detail as possible.

**Describe the bug**:
There is no information for air-gapped env deployment model

**Expected behaviour**:
at least offline or air gapped env deployment should be documented to include

all the necessary images + version
OCI is okay to clone offline. no issue

**Steps to reproduce the bug**:
try to deploy the logging operator offline or air gapped envs.

**Additional context**:

**Environment details**:
- Kubernetes version (e.g. v1.15.2):
- Cloud-provider/provisioner (e.g. AKS, GKE, EKS, PKE etc):
- logging-operator version (e.g. 2.1.1):
- Install method (e.g. helm or static manifests):
- Logs from the misbehaving component (and any other relevant logs):
- Resource definition (possibly in YAML format) that caused the issue, without sensitive data:

/kind bug

**Comment by pepov** [comment, marked as answer]
Hey! The images are documented here: https://kube-logging.dev/docs/image-versions/

Do you need more information than that? Feel free to join the project discord channel: https://discord.gg/sXFdDKEKK8
